docs/data/charts/bars/bars.md

```diff
@@ -165,21 +196,36 @@ Notice that the `onAxisClick` will handle both bar and line series if you mix th
 </ChartContainer>
 ```
 
-## Animation
+## Composition
 
-To skip animation at the creation and update of your chart, you can use the `skipAnimation` prop.
-When set to `true` it skips animation powered by `@react-spring/web`.
+TK
 
-Charts containers already use the `useReducedMotion()` from `@react-spring/web` to skip animation [according to user preferences](https://react-spring.dev/docs/utilities/use-reduced-motion#why-is-it-important).
+## Common examples
 
-```jsx
-// For a single component chart
-<BarChart skipAnimation />
+### Tiny Bar Chart
 
-// For a composed chart
-<ChartContainer>
-  <BarPlot skipAnimation />
-</ChartContainer>
-```
+{{"demo": "TinyBarChart.js"}}
```

Comment:
For context, those demo have been introduced to show the feature matching with [recharts](https://recharts.org/en-US/examples)

I'm not super attached to them. But I'm not in favor of merging those pages.

I would be more interested in keeping the page but with a dedicated design similar to [Echarts examples](https://echarts.apache.org/examples/en/index.html) a mosaic of preview the the option to open them.